client/apps/shop/src/app/pages/checkout/checkout-success/checkout-success.component.ts

```diff
@@ -1,13 +1,53 @@
-import {Component, OnInit, ChangeDetectionStrategy} from '@angular/core';
+import {
+  Component,
+  OnInit,
+  ChangeDetectionStrategy,
+  OnDestroy
+} from '@angular/core';
+import {CartService} from '../../../shared/services/cart/cart.service';
 
 @Component({
   selector: 'jfs-checkout-success',
   templateUrl: './checkout-success.component.html',
   styleUrls: ['./checkout-success.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class CheckoutSuccessComponent implements OnInit {
-  constructor() {}
+export class CheckoutSuccessComponent implements OnInit, OnDestroy {
+  constructor(public cartService: CartService) {}
 
-  ngOnInit() {}
+  result: {
```

Comment:
This is much better than just having `object` or `any`. However, if we had the same naming as the order object we could just do

```suggestion
  result: Partial<Order>
```

The `Partial` part is telling us that this might not contain everything from the order.